scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/core/SchedulingService.java

```diff
@@ -568,6 +568,36 @@ public Boolean call() throws Exception {
         }
     }
 
+    public boolean finishInErrorTask(final JobId jobId, final String taskName)
+            throws UnknownJobException, UnknownTaskException {
+        try {
+            if (status.isUnusable()) {
+                return false;
+            }
+
+            return infrastructure.getClientOperationsThreadPool().submit(new Callable<Boolean>() {
+                @Override
+                public Boolean call() throws Exception {
+                    TerminationData terminationData = jobs.finishInErrorTask(jobId, taskName);
+                    boolean taskfinished = terminationData.taskTerminated(jobId, taskName);
+                    submitTerminationDataHandler(terminationData);
+                    wakeUpSchedulingThread();
+                    return taskfinished;
+                }
+            }).get();
+        } catch (ExecutionException e) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Either log or rethrow this exception. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://master.ci.activeeon.com:9000/coding_rules#rule_key=squid%3AS1166)